### ISSUE TITLE
Make keymaps downloadable

### ISSUE THREAD

**Issue by mondalaci** [UltimateHackingKeyboard/agent#199]
This issue is needed to be implemented in order for us to crowdsource #91.

The idea is that there will be a download icon between the duplicate keymap and delete keymap icons on keymap pages. The user will be able to download the actual keymap by clicking on this icon. We could use [FileSaver.js](https://github.com/eligrey/FileSaver.js) for implementing the actual saving.

The filename should be "[KeymapName] keymap.json", so for example "QWERTY keymap.json".

The file should contain the JSON representation of the keymap inside of the following envelope:

```
{
    "site": "https://UltimateHackingKeyboard.com",
    "description": "Ultimate Hacking Keyboard keymap",
    "keyboardModel": "UHK60",
    "dataModelVersion": "{include the curent data model version here}",
    "objectType": "keymap",
    "objectValue": "{include the JSON representation of the keymap here}"
}
```

Let's not include Play Macro and Keymap Switch actions to the exported file now. We can include Play Macro actions later, but it's not needed for the time being.

**Comment by fjozsef** [comment]
See playground branch for similar saving implementation.

**Comment by fjozsef** [comment]
@mondalaci Shouldn't these keymaps also be importable? Otherwise the downloading is useless.

**Comment by mondalaci** [comment]
@fjozsef Eventually yes, but currently we only want to add a select few exported keymaps to the preset keymaps.

**Comment by fjozsef** [comment]
Blocked by #252 since the dataModelVersion is not accessible until then.
